cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -777,3 +777,24 @@ func TestImagePullCheck(t *testing.T) {
 		t.Fatalf("expected 2 errors but got %d: %q", len(errors), errors)
 	}
 }
+
+func TestNumCPUCheck(t *testing.T) {
+	var tests = []struct {
+		numCPU      int
+		numErrors   int
+		numWarnings int
+	}{
+		{0, 0, 0},
+		{int(^uint(0) >> 1), 1, 0},
```

Comment:
instead of `2^31-1` we might as well hardcode a very big int like 99999999, for better readability?
and so that we don't include `math`.

also having test names is a plus.